auth/services/selfsigned/web/templates/done_en.html

```diff
@@ -0,0 +1,21 @@
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Done</title>
+    <link rel="stylesheet" href="https://cdn.simplecss.org/simple.min.css">
```

Comment:
should be embed this, making it more resilient? Now the Nuts node will break when this source is slow/dead

Reply:
embedded for now